### PR TITLE
Disable multi-dimensional array test for UAP.

### PR DESCRIPTION
**Pull request by tijoytom**

### CODE REVIEW

**Comment by tijoytom** [comment]
@danmosemsft

File: src/Microsoft.CSharp/tests/ArrayHandling.cs [hunk 1]
```diff
@@ -31,25 +32,31 @@ public void ArrayTypeNames()
             d = new int[3];
             ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[]", ex.Message);
+        }
 
-            d = new int[3, 2, 1];
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Multi-dimensional array not supported")]
```
Comment (danmosemsft) [inline src/Microsoft.CSharp/tests/ArrayHandling.cs]: Can we include issue ID's
[SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Multi-dimensional array not supported": https://github.com/dotnet/corert/issues/3331")]

**Comment by danmosemsft** [comment]
build error.

File: src/Microsoft.CSharp/tests/ArrayHandling.cs [hunk 2]
```diff
@@ -31,25 +32,31 @@ public void ArrayTypeNames()
...
             Assert.Contains("int[,,]", ex.Message);
 
             d = Array.CreateInstance(typeof(int), new[] { 3, 2, 1 }, new[] { -2, 2, -0 });
             ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[,,]", ex.Message);
-
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corert/issues/3331")]
```
Comment (AtsushiKan) [inline src/Microsoft.CSharp/tests/ArrayHandling.cs]: Was this one really failing for that reason? I don't see any non-zero bounds or rank 1 multidims here.

File: src/Microsoft.CSharp/tests/ArrayHandling.cs [hunk 3]
```diff
@@ -31,25 +32,31 @@ public void ArrayTypeNames()
...
 
-            d = new int[3, 2, 1];
-            ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corert/issues/3331")]
+        public void MultiDimArrayTypeNames()
+        {
+            dynamic d = new int[3, 2, 1];
+            RuntimeBinderException ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[,,]", ex.Message);
```
Comment (AtsushiKan) [inline src/Microsoft.CSharp/tests/ArrayHandling.cs]: The first half of this test should work fine - it's using zero bounds. Better to the split the test and disable only the second half. Also, 3331 is about rank-1 multidims, not NZ bounds. The test you want to use is PlatformDetection.IsNonZeroLowerBoundArraySupported.

**Pull request merged**